types/react-native-fbsdk/index.d.ts

```diff
@@ -0,0 +1,1090 @@
+         */
+        lastRefreshTime: number;
+        /**
+         * Getter for the access token that is current for the application.
+         */
+        static getCurrentAccessToken(): Promise<AccessToken | null>;
+
+        /**
+         * Setter for the access token that is current for the application.
+         */
+        static setCurrentAccessToken(accessToken: AccessTokenMap): void;
+
+        /**
+         * Updates the current access token with up to date permissions,
+         * and extends the expiration date, if extension is possible.
+         */
+
+        static refreshCurrentAccessTokenAsync(): Promise<any>;
+
+        constructor(tokenMap: AccessTokenMap);
+
+        /**
+         * Gets the date at which the access token expires. The value is the number of
+         * milliseconds since Jan. 1, 1970, midnight GMT.
+         */
+        getExpires(): number;
+
+        /**
+         * Get the list of permissions associated with this access token. Note that the most up-to-date
+         * list of permissions is maintained by Facebook, so this list may be outdated if permissions
+         * have been added or removed since the time the AccessToken object was created. See
+         * https://developers.facebook.com/docs/reference/login/#permissions for details.
+         */
+        getPermissions(): Permissions[];
+
+        /**
+         * Gets the list of permissions declined by the user with this access token. It represents the
+         * entire set of permissions that have been requested and declined. Note that the most
+         * up-to-date list of permissions is maintained by Facebook, so this list may be outdated if
+         * permissions have been granted or declined since the last time an AccessToken object was
+         * created. See https://developers.facebook.com/docs/reference/login/#permissions for details.
+         */
+        getDeclinedPermissions(): Permissions[];
+
+        /**
+         * Gets the date at which the token was last refreshed. Since tokens expire, the Facebook SDK
+         * will attempt to renew them periodically. The value is the number of milliseconds since
+         * Jan. 1, 1970, midnight GMT.
+         */
+        getLastRefresh(): number;
+
+        /**
+         * Gets the ID of the Facebook Application associated with this access token.
+         */
+        getApplicationId(): string;
+
+        /**
+         * Gets user ID associated with this access token.
+         */
+        getUserId(): string;
+    }
+
+    /**
+     * Controls when an AppEventsLogger sends log events to the server
+     */
+    type AppEventsFlushBehavior =
+        /**
+         * Flush automatically: periodically (every 15 seconds or after every 100 events), and
+         * always at app reactivation. This is the default value.
+         */
+        'auto' |
+        /**
+         * Only flush when AppEventsLogger.flush() is explicitly invoked.
+         */
+        'explicity-only';
```

Comment:
Good spot, I copied it directly from the [flow type definition](https://github.com/facebook/react-native-fbsdk/blob/f27d02831c35a14790b40cc5c1c2381d480b028f/js/FBAppEventsLogger.js#L37).